test/unit/fact_parser_test.rb

```diff
@@ -225,6 +241,12 @@ class FactParserTest < ActiveSupport::TestCase
     assert_empty result[:attached_to]
     assert_empty result[:tag]
     assert result[:bridge]
+
+    result = parser.send(:set_additional_attributes, {}, 'br-ex')
+    assert result[:virtual]
+    assert_empty result[:attached_to]
+    assert_empty result[:tag]
+    assert result[:bridge]
```

Comment:
Hmm this could also solve the similar problem I have with NPAR NICs: https://projects.theforeman.org/issues/24073

They are named `em1_1` and `em1_2` but these are two different devices (they are actually hardware-virtual but we should not set virtual flag at all).

Reply:
yep)